src/components/demo/Demo.tsx

```diff
@@ -11,13 +13,47 @@ import Footer from 'components/general/footer/Footer';
 
 
 const Demo = () => {
+    /* Base variables */
+    const [inputDOI, setInputDOI] = useState<string>();
+
     return (
         <div className="d-flex flex-column min-vh-100 overflow-hidden">
             <Row>
                 <Header />
 
                 <Container fluid className={`${styles.content} pt-5`}>
-                    <Row className="mt-5">
+                    <Row className="my-4 justify-content-center">
+                        <Col className="col-md-auto">
+                            <h1 className="text-center"> DOI Tooltip Demo </h1>
+
+                            <p className="text-center">
+                                This demo page demonstrates the DOI Tooltip component offered by DiSSCo.
+                                <br />
+                                It is a component that can be used to cite DOI in text. When clicked on, it displays details on the DOI.
+                                <br />
+                                To implement the component yourself, please refer to this Notebook
+                                <br />
+                                <span className="c-accent">
+                                    <a href="https://colab.research.google.com/drive/1edY_tsIJvtQJsntTHk_xtnbdyuhLw6Br?usp=sharing"
```

Comment:
Is this a document on the DiSSCo drive or is it local? Might even be better to drop these documents somewhere in github

Reply:
Currently in the DiSSCo Drive, but will also include it here